broken/AceEditorHalogenHooks/src/Main.purs

```diff
@@ -33,7 +33,7 @@ _ace = SProxy :: SProxy "ace"
 containerComponent
   :: forall unusedQuery unusedInput unusedOutput anyMonad
    . MonadAff anyMonad
-  => H.Component HH.HTML unusedQuery unusedInput unusedOutput anyMonad
+  => H.Component unusedQuery unusedInput unusedOutput anyMonad
```

Comment:
I ran a `sed` command to remove `HH.HTML`
```
sed -i "s/Component HH\.HTML/Component/g" ./recipes/*HalogenHooks/*/Main.purs
```
but some recipes remained broken after that and other minimal attempts at fixing, which explains why some of these `HalogenHooks` recipes have the `HH.HTML` removed but still ended up in the `broken` directory. I think all the other recipes that ended up in `broken` were left unchanged.